scipy/linalg/_cython_signature_generator.py

```diff
@@ -179,16 +179,16 @@ def sigs_from_dir(directory, outfile, manual_wrappers=None, exclusions=None):
               # Signatures changed between 3.4.0 and 3.4.1.
               'dlasq5', 'slasq5',
```

Comment:
you can remove all this and below deprecations since we walked past 3.6.0 to 3.7.1 as min version.